jcl/src/openj9.jvm/share/classes/com/ibm/jvm/Dump.java

```diff
@@ -382,8 +382,8 @@ public static String systemDumpToFile(String fileNamePattern) throws InvalidDump
     		request = "system"; //$NON-NLS-1$
     	}
     	String dump = null;
-       	dump = triggerDump(request, "systemDumpToFile"); //$NON-NLS-1$
-       	return dump;
+	 	dump = triggerDump(request, "systemDumpToFile"); //$NON-NLS-1$
+	 	return dump;
```

Comment:
I'm not sure why you modified these two lines, but there's a strange mix of indentation styles:
* 384 starts with 4 spaces and a tab
* 385 & 386 start with a tab, a space and a tab

Maybe we should defer cleaning up that mess?

Reply:
Tried again, please see the latest update.